content/download/osx.adoc

```diff
@@ -13,16 +13,10 @@ KiCad is known to work on Mac OS X 10.11 through macOS 10.14.  It may work on ne
 
 == Stable Release
 
-Current Version: *5.0.2*
+Current Version: *5.0.2-5*
 
-[NOTE]
-For the time being there have been some packaging related issues with
-5.0.2 for macOS, so please get 5.0.1 from
-https://kicad-downloads.s3.cern.ch/index.html?prefix=osx/stable/[https://kicad-downloads.s3.cern.ch/index.html?prefix=osx/stable/]
-or hold back upgrading on macOS.
-
-//- For macOS 10.14: https://kicad-downloads.s3.cern.ch/osx/stable/kicad-unified-5.0.2-10_14.dmg[KiCad] http://www2.futureware.at/~nickoe/kicad-downloads-mirror/osx/stable/kicad-unified-5.0.2-10_14.dmg[[mirror]]
-//- For macOS 10.11-10.13: https://kicad-downloads.s3.cern.ch/osx/stable/kicad-unified-5.0.2.dmg[KiCad] http://www2.futureware.at/~nickoe/kicad-downloads-mirror/osx/stable/kicad-unified-5.0.2.dmg[[mirror]]
+//- For macOS 10.14: https://kicad-downloads.s3.cern.ch/osx/stable/kicad-unified-5.0.2-5-10_14.dmg[KiCad] http://www2.futureware.at/~nickoe/kicad-downloads-mirror/osx/stable/kicad-unified-5.0.2-5-10_14.dmg[[mirror]]
```

Comment:
Hmm, the file actually uploaded is named with ...10.14.dmg, shouldn't it be 10_14 as here?  See the difference? The period vs the underscore. I don't really care too much what it is called. You can decide.